test/fuzz/main/README.md

```diff
@@ -0,0 +1,658 @@
+also would ensure that cluster will respond with correct results to _any_ allowed read query.
+
+To begin specifying operations for a new partition, either start calling methods on the `HistoryBuilder`, or, if you 
+would like to specify the partition which Harry needs to visit use `#visitPartition` or `#beginBatch` have to be called, 
+for starting a visit to a particular partition with a single or multiple actions.
+
+After that, the actions are self-explanatory: `#insert`, `#update`, `#deleteRow`, `#deleteColumns`, `#deleteRowRange`, `#deleteRowSlice`
+`#deletePartition`.
+
+After history generated by `HistoryBuilder` is replayed using `ReplayingVisitor` (or by using a `ReplayingHistoryBuilder` 
+which combines the two for your convenience), you can use any model (`QuiescentChecker` by default) to validate queries.
+Queries can be provided manually or generated using `QueryGenerator` or `TypedQueryGenerator`.
+
+# Basic Terminology
+
+* Inflate / inflatable: a process of producing a value (for example, string, or a blob) from a `long` descriptor that
+  uniquely identifies the value. See [data generation](https://github.com/apache/cassandra-harry#data-generation)
+  section of this guide for more details.
+* Deflate / deflatable: a process of producing the descriptor the value was inflated from during verification.
+  See [model](https://github.com/apache/cassandra-harry#model) section of this guide for more details.
+
+For definitions of logical timestamp, descriptor, and other entities used during inflation and deflation, refer
+to [formal relationships](https://github.com/apache/cassandra-harry#formal-relations-between-entities) section.
+
+# Features
+
+Currently, Harry can exercise the following Cassandra functionality:
+
+* Supported data types: `int8`, `int16`, `int32`, `int64`, `boolean`, `float`, `double`, `ascii`, `uuid`, `timestamp`.
+  Collections are only _inflatable_.
+* Random schema generation, with an arbitrary number of partition and clustering keys.
+* Schemas with arbitrary `CLUSTERING ORDER BY`
+* Randomly generated `INSERT` and `UPDATE` queries with all columns or arbitrary column subset
+* Randomly generated `DELETE` queries: for a single column, single row, or a range of rows
+* Inflating and validating entire partitions (with allowed in-flight queries)
+* Inflating and validating random `SELECT` queries: single row, slices (with single open end), and ranges (with both
+  ends of clusterings specified)
+
+Inflating partitions is done
+using [Reconciler](https://github.com/apache/cassandra-harry/blob/master/harry-core/src/harry/reconciler/Reconciler.java).
+Validating partitions and random queries can be done
+using [Quiescent Checker](https://github.com/apache/cassandra-harry/blob/master/harry-core/src/harry/model/QuiescentChecker.java)
+and [Exhaustive Checker](https://github.com/apache/cassandra-harry/blob/master/harry-core/src/harry/model/ExhaustiveChecker.java).
```

Comment:
nit: I guess we can point to ourselves now instead of `cassandra-harry`?

Reply:
I've removed the navigation references for now. I think it should be straightforward to search for sections.